packages/shared/src/functions.ts

```diff
@@ -13,3 +13,17 @@ export const bigintMin = (...args: bigint[]): bigint =>
 
 export const bigintMax = (...args: bigint[]): bigint =>
   args.reduce((max, current) => (current > max ? current : max));
+
+export const ONE_IN_PIP = 10000;
+export const ONE_IN_HUNDREDTH_PIPS = ONE_IN_PIP * 100;
+
+export const calculatePipAmountFromAmount = (
```

Comment:
```suggestion
export const getPipAmountFromAmount = (
```
just a suggestion, you can ignore if you don't like it